Change old name references in installation instructions (#25)
ba-st/Ansible: docs/Installation.md

You can load **Ansible** evaluating:
```smalltalk
Metacello new
	baseline: 'Ansible';

> 	repository: 'github://fortizpenaloza/Ansible:<DEFAULT_BRANCH>/source';

Reviewer: ```suggestion
	repository: 'github://ba-st/Ansible:release-candidate/source';
```

	load.
```

> >  Change `<DEFAULT_BRANCH>` to some released version if you want a pinned version

Reviewer: ```suggestion
>  Change `release-candidate` to some released version if you want a pinned version
```

setUpDependencies: spec

	spec
		baseline: 'Ansible'
			with: [ spec

> 				repository: 'github://fortizpenaloza/Ansible:v{XX}/source';

Reviewer: ```suggestion
				repository: 'github://ba-st/Ansible:v{XX}/source';
```
